apps/web/app/not-found.tsx

```diff
@@ -1,12 +1,210 @@
-import React from "react";
+import { getFixedT } from "app/_utils";
+import { headers } from "next/headers";
+import Link from "next/link";
+
+import { getServerSessionForAppDir } from "@calcom/feature-auth/lib/get-server-session-for-app-dir";
+import {
+  getOrgDomainConfigFromHostname,
+  subdomainSuffix,
+} from "@calcom/features/ee/organizations/lib/orgDomains";
+import { DOCS_URL, IS_CALCOM, WEBSITE_URL } from "@calcom/lib/constants";
+import { Icon } from "@calcom/ui";
+
+enum PageType {
```

Comment:
Consider aligning the string values with the enum keys for consistency:

```
enum PageType {
    ORG = "ORG",
    TEAM = "TEAM",
    USER = "USER",
    OTHER = "OTHER",
}
```

Reply:
I agree, thanks :)

Reply:
Thanks for taking your time to look through our PRs btw !

Reply:
My pleasure ! Keep up the good work :)